helm/default-apps-aws/values.yaml

```diff
@@ -67,7 +67,19 @@ userConfig:
           prefix: "/registry/"
         events:
           prefix: "/registry/events/"
-
+  securityBundle:
+    configMap:
+      values:
+        userConfig:
+          kyverno:
+            configMap:
+              values:
+                kyverno:
+                  admissionController:
+                    tolerations:
+                    - key: "node.cluster.x-k8s.io/uninitialized"
```

Comment:
This applies to all CAPI implementations. Should we have this in the `config` repo instead?

Reply:
Why the config repo? If we want to apply it everywhere we'd need to add it to the security bundle or kyverno itself. 
I don't know if we _should_ add it though as I've only tested this on CAPA and I don't know what the consequences would be on other providers.

Reply:
`config` repo is MC only so that does not make sense